File: test/py/case61_inheritance.py
class Base61:
    def value(self) -> int:
        return 61


class Child61(Base61):
    def value2(self) -> int:
        return self.value() + 1


if __name__ == "__main__":
    c: Child61 = Child61()
    print(c.value2())

File: test/cs/case61_inheritance.cs
using System;

public static class Program
{
    public class Base61
    {
        public int value()
        {
            return 61;
        }
    }

    public class Child61 : Base61
    {
        public int value2()
        {
            return (this.value() + 1);
        }
    }

    public static void Main(string[] args)
    {
        Child61 c = new Child61();
        Console.WriteLine(c.value2());
    }
}
